package.json

```diff
@@ -64,8 +64,8 @@
     },
     "peerDependencies": {
         "@prefecthq/prefect-design": "^2.0.8",
-        "@prefecthq/vue-charts": "^2.0.2",
-        "@prefecthq/vue-compositions": "^1.5.9",
+        "@prefecthq/vue-charts": "^2.0.3",
+        "@prefecthq/vue-compositions": "^1.6.2",
```

Comment:
I'm using a new utility from vue-compositions@1.6.2 so ui-library now requires a minimum version of 1.6.2

separately but related, vue-charts@2.0.2 had peerDependencies: { @prefecthq/vue-compositions: 1.5.9 } pinned exactly whereas 2.0.3 does a looser version constraint so I'm going ahead and bumping that here as well. 🤔 

Recommend checking out the package-lock.json changes - they're pretty minimal but noting that they're there.